net/mscp/Portfile

```
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
```

Since this is a Github project, you can also use the `github` portgroup.

```suggestion
PortGroup           cmake 1.1
PortGroup           github 1.0
```